src/components/game/coreState/Piece.js

```diff
@@ -36,29 +63,25 @@ class Piece {
     if (randint(0, 2) == 1) {
       this.flip();
     }
-
-    // Set this piece's color based on its initial direction; there will be more
-    // room to customize this.
-
-    // TODO: Define Colors in a more appropriate class and keep it in RenderProps with GameState
-    // when that time comes/
-    this.color = new Color({ red: 100, green: 100, blue: 100 });
   }
 
   // Just for formality/convention, we do this each time we move something from the game
   // back to the stage.
   unmountPiece() {
     this.mounted = false;
-    this.cx = undefined;
-    this.cy = undefined;
+    this.cx = 0;
+    this.cy = 0;
     this.dxn = undefined;
     this.pidSie = undefined;
   }
 
   // The function to fill the coreState with cells corresponding to this Piece; this will
   // be used for cases like the render script accessing the color in the parents
   createCell() {
```

Comment:
Take a closer look at createCell() there might be something a bit off with initializing new Cells